libraries/botbuilder/tests/teamsInfo.test.js

```diff
@@ -936,11 +957,11 @@ describe('TeamsInfo', function () {
                 );
             });
 
-            it(`should error if the adapter doesn't have a createConnectorClient method`, function () {
-                assert.throws(
-                    () => TeamsInfo.getConnectorClient({ adapter: {} }),
-                    new Error('This method requires a connector client.')
-                );
+            it(`should fallback to the connectorClient on turnState if adapter doesn't exist in context.adapter`, function () {
+                const context = new TurnContext({});
+                context.turnState.set(context.adapter.ConnectorClientKey, connectorClient);
+                const result = TeamsInfo.getConnectorClient(context);
+                assert.deepStrictEqual(result, connectorClient);

```

Comment:
I think `strictEqual` is what you want, effectively `===` which ensures the references are exactly equivalent.

Reply:
Changed. For clarification, the switch is saying, "I care that the parent objects are exactly equivalent and don't need to check each child object"?

Reply:
It looks like [`deepStrictEqual` defers to reference equality](https://github.com/nodejs/node/blob/fa0c6883a65737e0135b107f5e886acbdfb89ec5/lib/internal/util/comparisons.js#L132) which is why this was working. I think it's better to be clear in intent - `deepStrictEqual`, to me, means that you are recursing into objects. `strictEqual`, on the other hand, is a `===` which is what you are wanting to verify here.